client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -33,8 +33,9 @@ class PublicizeConnection extends Component {
 	};
 
 	render() {
-		const { service_name: name, label, disabled, display_name } = this.props.connectionData;
+		const { service_name: name, disabled, display_name, unique_id } = this.props.connectionData;
 		const { connectionOn } = this.props;
+		const label = 'connection-' + name + '-' + unique_id;
```

Comment:
Could `label` be renamed to `id` or something, since as such it's a bit misleading?

It's also used as `title={ label }` and thus shows up in screen readers and when hovering the label:

<img width="277" alt="image" src="https://user-images.githubusercontent.com/87168/48334023-f287c380-e661-11e8-94b7-faf848bf47ad.png">

Reply:
Good catch! Both addressed in 511ec7c595f749a5d1c505d9371390392ab42503

Also, it seems we now have an ID for the label, so I removed the a11y eslint rule disabling in 50ccb9a7c42d6782bcb73c89b89d74127b119100.

This could use another final look before I ship, @simison, thanks!